Jellyfin.Server.Implementations/Devices/DeviceManager.cs

```diff
@@ -37,6 +39,24 @@ public DeviceManager(IDbContextFactory<JellyfinDbContext> dbProvider, IUserManag
         {
             _dbProvider = dbProvider;
             _userManager = userManager;
+            _devices = new ConcurrentDictionary<int, Device>();
+            _deviceOptions = new ConcurrentDictionary<string, DeviceOptions>();
+
+            using var dbContext = _dbProvider.CreateDbContext();
+            foreach (var device in dbContext.Devices
+                         .Include(d => d.User)
```

Comment:
I think this has a chance to get out of sync with the User cache. Looks like only the username is actually used?

User might also get updated when the devices is updated, I haven't checked.

Reply:
This is a required relationship, but most of our logic does not depend on this yet. For current use case I think it would be good enough to not include this and query the user from user cache directly, as the userId should not change.